src/wallet/api/wallet.h

```diff
@@ -86,8 +86,8 @@ class WalletImpl : public Wallet
     ConnectionStatus connected() const;
     void setTrustedDaemon(bool arg);
     bool trustedDaemon() const;
-    uint64_t balance() const;
-    uint64_t unlockedBalance() const;
+    uint64_t balance(uint32_t account_index) const;
```

Comment:
The full set of API is declared in walle2_api.h where there are three functions `balance()`, `balance(index)` and `balanceAll()` which return the balance of the currently selected account, the balance of the account of the given index, and the total sum of the balances across all accounts, respectively.